packages/eslint-plugin/src/configs/eslint-recommended.ts

```diff
@@ -23,7 +23,7 @@ export = {
         'no-this-before-super': 'off', // ts(2376)
         'no-undef': 'off', // ts(2304)
```

Comment:
```suggestion
        'no-undef': 'off', // ts(2304) & ts(2552)
```

https://github.com/eslint/eslint/pull/17500#discussion_r1311670674